CMSgov/easi-app
# [NOREF] Removed context from the TRB seederConfig struct to conform with Go recommendation

# NOREF

## Changes and Description

I was reading some [go docs on the context package](https://pkg.go.dev/context) and found this quote:
```Do not store Contexts inside a struct type; instead, pass a Context explicitly to each function that needs it. The Context should be the first parameter, typically named ctx```

The TRB seeding code breaks this best practice, so I fixed that in this PR.



## How to test this change
Run the seed script with `./scripts/dev db:seed` and confirm that it's seeding the data correctly.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
